g2p_program_reimbursement/views/program_view.xml

```diff
@@ -4,6 +4,22 @@
         <field name="domain">[("is_reimbursement_program", "=", False)]</field>
     </record>
 
+    <!-- Tree view Inheritance for reimbursement-->
+    <record id="view_program_list_reimbursement_tree" model="ir.ui.view">
+        <field name="name">view_program_reimbursement_list</field>
+        <field name="model">g2p.program</field>
+        <field name="priority" eval="103" />
+        <field name="inherit_id" ref="g2p_programs.view_program_list_tree" />
+        <field name="arch" type="xml">
+            <field name="eligible_beneficiaries_count" position="attributes">
```

Comment:
We can hide this column, rather than renaming.. The count here doesn't serve much purpose here,

Reply:
Are the modifications made to both the programs and the reimbursement programme views?